apps/server/src/modules/management/uc/database-management.uc.ts

```diff
@@ -145,6 +148,41 @@ export class DatabaseManagementUc {
 		return allCollectionsWithFilePaths;
 	}
 
+	private async dropCollectionIfExists(collectionName: string) {
+		const collectionExists = await this.databaseManagementService.collectionExists(collectionName);
+		if (collectionExists) {
+			// clear existing documents, if collection exists
+			await this.databaseManagementService.clearCollection(collectionName);
+		} else {
+			// create collection
+			await this.databaseManagementService.createCollection(collectionName);
+		}
+	}
+
+	async seedDatabaseCollectionsFromFactories(collections?: string[]): Promise<string[]> {
+		const seededCollectionsWithAmount = await Promise.all(
+			generateSeedData((s: string) => this.injectEnvVars(s))
```

Comment:
Can we put this block to const first?
const promises = generateSeedData(..)
const seededCollectionsWithAmount = await Promise.all(promises);